examples/lilygo-t5-47-epd-platformio/platformio.ini

```diff
@@ -0,0 +1,39 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[common_env_data]
+framework = arduino
+board_build.f_cpu = 80000000L
```

Comment:
I'm usually using 240 MHz, but I'm not sure if taking longer to process or running at a slower speed outweighs the other here energy-wise.

Reply:
I just set it to 240 MHz too now. I don't think it has a substantial impact on energy consumption.
Honestly I just use 80 MHz because it's default in many examples and I never have problems with slow speeds.
For this epd driving it could be beneficial